opentelemetry-sdk/src/opentelemetry/sdk/_logs/export/__init__.py

```diff
@@ -293,6 +299,9 @@ def emit(self, log_data: LogData) -> None:
         """
         if self._shutdown:
             return
+        if self._pid != os.getpid():
```

Comment:
Curious as to why this is being done on `emit` rather than when the processor is instantiated?

Reply:
Yes, that's reasoning behind keeping it here.

Reply:
@lzchen please mark as resolved so that we can merge :v: